lib/cchain/queryutil/staking.go

```diff
@@ -99,7 +100,7 @@ func DelegatorInflationRates(ctx context.Context, cprov cchain.Provider, delegat
 // DelegationBalance represents the total delegation balance of a delegator.
 type DelegationBalance struct {
 	DelegatorAddress sdk.AccAddress
-	Balance          sdk.Coin
+	Balance          big.Int
```

Comment:
i know it's a little weird but with `big.Int` you generally just work with pointers.

So this should be `Balance *big.Int`.